.github/workflows/ci.yml

```diff
@@ -66,6 +66,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
```

Comment:
Need to use v1.2
The author never updated the v1 tag.

Reply:
I thought the vX tag always picked up the latest minor on the given X major 🤔

Reply:
Yes, you are correct. No need to specify a minor version here, just keep v1.

Reply:
> I thought the vX tag always picked up the latest minor on the given X major 🤔

It's just a convention afaik. In that repo v1 wasn't updated.

Reply:
In git, you normally don't update tags. The convention in hendrikmuhs/ccache-action matches the GitHub Actions convention. Currently, the latest tag is `v1.2.2`, so specifying `v1` will resolve to that tag.

Reply:
Well at least Github's own documentation tells you to move the major version tags:
https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management
https://github.com/actions/toolkit/blob/main/docs/action-versioning.md#recommendations

Reply:
You're right, I just had the time to check the docs today. There is an issue on their repo asking for them to move the major tag, let's see if we get a reply there: https://github.com/hendrikmuhs/ccache-action/issues/111

Reply:
Since it seems they aren’t going to update their v1 tag, let’s go with @Trass3r ’s suggestion here and use the v1.2 tag.

Reply:
Note we also need to tinker with msbuild to use cache, like suggested on another comment.

That being said, we might want to bump up the ccache on all other actions in the meantime.